antiplag/tasks.py

```diff
@@ -39,8 +41,15 @@ def process_documents(submission_id):
 
     # update submission status
     submission.status = Submission.SubmissionStatus.PROCESSED
-    submission.save()
 
+    # send email when done
+    send_mail('Tvoja kontrola bola dokončená',
+        'Pozri si výsledok svojej kontroly na: antiplag.sk/result/' + str(submission.id) + "/" ,
+        'antiplag@antiplag.sk',
+        [submission.email],
```

Comment:
What if the submission.email is NULL? Is this case handled by django core or will it throw an error?